Docs: Add documentation with regards to the new prettify function (#36856)
grafana/grafana: docs/sources/explore/logs-integration.md

Set this to True if you want the display to use line wrapping. If set to False, it will result in horizontal scrolling.

#### Prettify JSON

> Set this to true if you want to pretty print all JSON logs. This will not affect logs in any format other than JSON.

Reviewer: ```suggestion
Set this to `true` to pretty print all JSON logs. This setting does not affect logs in any format other than JSON.
```
